content/blog/kallisti5/2021-03-22_feb_2021_survey_results.md

```diff
@@ -0,0 +1,138 @@
++++
+type = "blog"
+title = "Feb 2021 - Survey Results"
+author = "kallisti5"
+date = "2021-03-22 17:37:59-05:00"
+tags = ["haiku", "software", "survey"]
++++
+
+Earier this month, I created a quick survey to guage what topics are most important to the Haiku communtity.
+We had 387 responses making this pretty substantial in helping us evaluate places we need to improve.<br/><br/>
+Thanks to everyone who filled out this survey!<br/>
+I'll be going over the results and adding a little commentary along the way. Lets go!
+
+## Which version of Haiku do you normally use?
+
+<img src="/files/survey_feb2021/1.png" alt="59.7% Stable, 40.3% Nightly"><br/>
+
+So far, not too suprising. A lot of people value the stable R1/Beta 2 release, but most prefer the nightly images. This makes sense as R1 / Beta 2 was released around 9 months ago.
```

Comment:
your alt text says the opposite (60% using the stable release)

Reply:
fixed